libs/modelviewer/src/ModelViewer.tsx

```diff
@@ -30,25 +32,25 @@ export const FusionModelViewer = (props: PropsWithChildren<FusionModelViewerProp
             props.options?.fallbackComponent ? props.options.fallbackComponent : Message
           }
         >
-          <ModelViewer {...props} />
+          <ModelViewerContainer {...props} />
         </MessageBoundary>
       </ConfigContextProvider>
     </QueryClientProvider>
   );
 };
 
-const ModelViewer = ({
+const ModelViewerContainer = ({
```

Comment:
The naming confuses me, the ModelViewer renders the ModelViewerContainer?. Doesnt container imply that its a parent or ancestor?

Reply:
Fixed.
Refactored to ModelViewerContent